### PR TITLE
Translate DOM options part within migration file to Spanish

### PR DESCRIPTION
**Pull request by FLarra**

### CODE REVIEW

File: src/v2/guide/migration.md [hunk 1]
```diff
@@ -1462,22 +1462,22 @@ Use the [Vue Devtools](https://github.com/vuejs/vue-devtools) for the optimal de
...
-``` js
+```js
 new Vue({
   el: '#app',
   template: '<div id="app"> ... </div>'
 })
 ```
 
-Or with a render function:
+O una función render:
```
Comment (Tintef) [inline src/v2/guide/migration.md]: What do you think about changing this to: `O una función de renderizado:`

Comment (FLarra) [reply src/v2/guide/migration.md]: I thought about that, but then thought that the function is called `render` so maybe was better to keep it in the same way (?)

I'm not sure about what is the best option here

Comment (Tintef) [reply src/v2/guide/migration.md]: Maybe we can wrap "render" between backticks 🤔 (``O una función de `render`:``)

Comment (sxcamacho) [reply src/v2/guide/migration.md]: for me `O con una función de renderizado:` is okay

File: src/v2/guide/migration.md [hunk 2]
```diff
@@ -1492,8 +1492,10 @@ new Vue({
 
 {% raw %}
 <div class="upgrade-path">
-  <h4>Upgrade Path</h4>
-  <p>Run the <a href="https://github.com/vuejs/vue-migration-helper">migration helper</a> on your codebase to find examples of <code>replace: false</code>.</p>
+  <h4>Ruta de actualización</h4>
+  <p>
+    Ejecute el <a href="https://github.com/vuejs/vue-migration-helper">asistente de migración</a> en su código para entontrar ejemplos de <code>replace: false</code>.
```
Comment (sxcamacho) [inline src/v2/guide/migration.md]: Typo: `entontrar` should be `encontrar`

**Comment by FLarra** [comment]
@sxcamacho @Tintef thanks for the reviews, just addressed your comments :) Let me know what you think

**Review by sxcamacho — APPROVED** [review]

**Pull request merged**